synfig-core/src/synfig/rendering/common/task/taskcontour.h

```diff
@@ -53,7 +53,7 @@ class TaskContour: public Task, public TaskInterfaceTransformation
 	virtual Token::Handle get_token() const { return token.handle(); }
 
 public:
-	Contour::Handle contour;
+	std::shared_ptr<Contour> contour;
```

Comment:
I tried to use `std::unique_ptr`, but couldn't compile because some Task copy calls... I didn't investigate it though

Reply:
I didn't remember, but maybe you know - if the Task is always unique, or copies are possible?

Reply:
Hm... They can be set as sub_task of one or more tasks...

Reply:
But contour itself I don't know if it is shared to other stuff besides the task.